### PR TITLE
added driver fetch size

### PR DESCRIPTION
**Pull request by conker84**
added driver fetch size

### CODE REVIEW

File: common/src/main/scala/org/neo4j/spark/util/Neo4jOptions.scala [hunk 1]
```diff
@@ -301,6 +303,7 @@ case class Neo4jDriverOptions(
     if (acquisitionTimeout > -1) builder.withConnectionAcquisitionTimeout(acquisitionTimeout, TimeUnit.MILLISECONDS)
     if (livenessCheckTimeout > -1) builder.withConnectionLivenessCheckTimeout(livenessCheckTimeout, TimeUnit.MILLISECONDS)
     if (connectionTimeout > -1) builder.withConnectionTimeout(connectionTimeout, TimeUnit.MILLISECONDS)
+    if (fetchSize > -1) builder.withFetchSize(fetchSize)
```
Comment (fbiville) [inline common/src/main/scala/org/neo4j/spark/util/Neo4jOptions.scala]: beware that `-1` is actually a valid value for the Java driver.
> By default, the value is set to 1000. Use -1 to disables back pressure and config client to pull all records at once after each run.

Comment (conker84) [reply common/src/main/scala/org/neo4j/spark/util/Neo4jOptions.scala]: Done

**Review by fbiville — APPROVED** [review]

**Comment by fbiville** [comment]
Let's close this PR for now. We can reopen it if we ever think it makes sense to have it.

**Comment by conker84** [comment]
@fbiville Do you think that cannot be useful for heavy operations? As it's basically ready, for me is just an added value to the connector.

**Comment by fbiville** [comment]
Let's discuss this internally and re-evaluate.
There could be value indeed, but no-one has asked for it yet and expanding the configuration surface means more choice and potentially more confusion for the end user too.

**Pull request closed without merging**